test/test-helper.js

```diff
@@ -1033,8 +1033,6 @@ const helper = {
              * Shutdowns the hosts and invoke the optional callback.
              */
             shutdown: function (cb) {
-                hosts.forEach((h) => h.shutdown(false));
-
                 if (cb) {
                     cb();
                 }
```

Comment:
:thinking: I though we do not support shutdown at all. The whole thing should therefore be deleted, shouldn't it?

Reply:
OK; what I mean is that if we removed the code that actually shuts hosts down (and its caller at 1036), we should remove the remaining logic 1035-1038+ too.

Reply:
Don't you agree?